### PR TITLE
soupault: add page

### PR DESCRIPTION
**Pull request by dmbaturin**
Full disclosure: I'm the maintainer of soupault. 

- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):** 4.2.0

### CODE REVIEW

File: pages/common/soupault.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
+# soupault
+
+> Soupault is a static website generator based on HTML element tree rewriting.
+> It can also be used as an HTML post-processor or metadata extractor.
+> More information: <https://soupault.app>.
+
+- Initialize a minimal website project:
```
Comment (mfrw) [inline pages/common/soupault.md:7]: How about we add the `current dir` ?
```suggestion
- Initialize a minimal website project in the current working directory:
```

File: pages/common/soupault.md [hunk 2]
```diff
@@ -0,0 +1,25 @@
...
+
+`soupault --init`
+
+- Build a website:
+
+`soupault`
+
+- Override default config file and directory locations:
+
+`soupault --config {{config_path}} --site-dir {{input_dir) --build-dir {{output_dir}}`
```
Comment (kbdharun) [inline pages/common/soupault.md:17]:
```suggestion
`soupault --config {{config_path}} --site-dir {{input_dir}} --build-dir {{output_dir}}`
```

**Review by kbdharun — COMMENTED** [review]
Thanks for your contribution. I have a small suggestion for this page.

**Review by CleanMachine1 — APPROVED** [review]
lgtm after suggested changes.

**Review by kbdharun — APPROVED** [review]

**Pull request merged**